README.md

```diff
@@ -114,6 +114,22 @@ It is possible to generate spdx based SBOM (software bill of materials)
 documents as part of the build. Just pass `--with-sbom` and
 it will put them into the output directory.
 
+### Cloud integration
+
+When building an image type that can be uploaded to the cloud
+(e.g. an "ami") image-builder will automatically upload if
+all cloud paramters are provided, e.g.
+```
+$ imge-builder build ami --distro centos-9 \
+    --aws-region=us-east-1 \
```

Comment:
I wonder why you chose `--key=value` notation here when the `--distro` flag one line above uses `--key value`. I think we should be consistent.